packages/std/src/exports.rs

```diff
@@ -20,12 +20,18 @@ use crate::ibc::{
 };
 use crate::imports::{ExternalApi, ExternalQuerier, ExternalStorage};
 use crate::memory::{alloc, consume_region, release_buffer, Region};
+#[cfg(feature = "abort")]
+use crate::panic::install_panic_handler;
 use crate::query::CustomQuery;
 use crate::results::{ContractResult, QueryResponse, Reply, Response};
 use crate::serde::{from_slice, to_vec};
 use crate::types::Env;
 use crate::{CustomMsg, Deps, DepsMut, MessageInfo};
 
+#[cfg(feature = "abort")]
```

Comment:
We don't need this at all.

This is a request from the Go/Wasmd runtime that it provides a feature, something which is passed through between the Wasm contract and the Go runtime.

This is not the case here